sdk/storage/storage-file-datalake/package.json

```diff
@@ -57,6 +57,10 @@
     "dist/",
     "dist-esm/storage-file-datalake/src/",
     "dist-esm/storage-common/src/",
+    "dist-esm/storage-blob/src/policies/",
```

Comment:
Instead of adding more files into this package, can we import the necessary files from `@azure/storage-blob` or are they hidden / not publicly exposed? We already have a dependency on `@azure/storage-blob` in this package

Reply:
The types are not publicly exposed, we'd need to import file for now. We do the same for storage-file-share and storage-queue packages.